bin/cluster_dev.sh

```diff
@@ -23,15 +23,27 @@ fi
 
 INNER_SHELL=${SHELL:-"/bin/bash"}
 
+echo "----- Faasm development cluster -----"
+
 docker-compose -f docker-compose.yml \
     up \
     --no-recreate \
-    -d \
-    faasm-cli
+    -d 
+
+if [[ "$1" == "faasm" ]]; then
+    CLI_CONTAINER="faasm-cli"
+elif [[ "$1" == "cpp" ]]; then
+    CLI_CONTAINER="cpp"
+else
+    echo "Not starting a CLI container"
+    exit 0
+fi
```

Comment:
I found we need to run both containers when developing against the local cluster.